datasets/multi_booked/multi_booked.py

```diff
@@ -0,0 +1,166 @@
+                },
+            ),
+        ]
+
+    def _generate_examples(self, dirpath):
+        for id_, filepath in enumerate(sorted(Path(dirpath).iterdir())):
+            example = defaultdict(lambda: defaultdict(list))
+            with open(filepath, encoding="utf-8") as f:
+                try:
+                    for _, elem in ET.iterparse(f):
+                        if elem.tag == "text":
+                            for child in elem:
+                                example["text"]["wid"].append(child.attrib.get("wid", ""))
+                                example["text"]["sent"].append(child.attrib.get("sent", ""))
+                                example["text"]["para"].append(child.attrib.get("para", ""))
+                                example["text"]["word"].append(child.text)
+                        elif elem.tag == "terms":
+                            for child in elem:
+                                example["terms"]["tid"].append(child.attrib.get("tid", ""))
+                                example["terms"]["lemma"].append(child.attrib.get("lemma", ""))
+                                example["terms"]["morphofeat"].append(child.attrib.get("morphofeat", ""))
+                                example["terms"]["pos"].append(child.attrib.get("pos", ""))
+                                targets = []
+                                for target in child.findall("span/target"):
+                                    targets.append(target.get("id", ""))
+                                example["terms"]["target"].append(targets)
+                        elif elem.tag == "opinions":
+                            for child in elem:
+                                example["opinions"]["oid"].append(child.attrib.get("oid", ""))
+                                # Opinion holder
+                                opinion_holder = child.find("opinion_holder")
+                                targets = []
+                                for target in opinion_holder.findall("span/target"):
+                                    targets.append(target.get("id", ""))
+                                example["opinions"]["opinion_holder_target"].append(targets)
+                                # Opinion target
+                                opinion_target = child.find("opinion_target")
+                                targets = []
+                                for target in opinion_target.findall("span/target"):
+                                    targets.append(target.get("id", ""))
+                                example["opinions"]["opinion_target_target"].append(targets)
+                                # Opinion expression
+                                opinion_expression = child.find("opinion_expression")
+                                example["opinions"]["opinion_expression_polarity"].append(
+                                    opinion_expression.attrib.get("polarity", "")
```

Comment:
```suggestion
                                    opinion_expression.attrib["polarity"]
```
if there are no missing values, can you remove the `.get` ?
same for the other fields

Reply:
@lhoestq done!